### PR TITLE
Added config file

### PR DESCRIPTION
**Pull request by Anilreddy999**
Please review. As part of Matomo setup, adding configuration file to environment.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,14 +3,17 @@ FROM matomo:3.10.0
...
 RUN chmod 775 /entrypoint.sh
+RUN chmod 775 /matomo-entrypoint.sh
 
 RUN sed -i -e 's/:80/:9000/g' /etc/apache2/sites-enabled/000-default.conf
 RUN sed -i -e 's/Listen 80/Listen 9000/g' /etc/apache2/ports.conf
 
 USER ${USERMAP_UID}
 
-ENTRYPOINT ["/entrypoint.sh"]
+ENTRYPOINT ["/entrypoint.sh", "/matomo-entrypoint.sh"]
```
Comment (tomfitzherbert) [inline Dockerfile]: We can only specify a single ENTRYPOINT script. Any values following the script filepath will be treated as parameters.

File: matomo-entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+set -e
+
+sed -i -e "s/{MATOMO_MYSQL_HOST}/{{.MATOMO_MYSQL_HOST}}/" /var/www/html/config/config.ini.php
```
Comment (tomfitzherbert) [inline matomo-entrypoint.sh]: As discussed. This needs a bit of an update.

File: Dockerfile (hunk 1 above)
Comment (Anilreddy999) [reply Dockerfile]: Merged two entry point scripts into single entry point script

File: matomo-entrypoint.sh (hunk 2 above)
Comment (Anilreddy999) [reply matomo-entrypoint.sh]: corrected syntaxes

**Pull request merged**